### ISSUE TITLE
Neutral Site flag not used/working

### ISSUE THREAD

**Issue by Spartan97** [Spartan97/JCS#1]
Sports-references does not have location data for games, so neutral site games are still listed as Home/Away. This data could possibly be pulled from ESPN (want to avoid because of how often the UI changes), or I could make a tool (think gamelog with checkboxes) to manually update this quickly.

**Comment by jkeena88** [comment]
Don't worry about pulling it from another source. I'm not sure how hard it would be to make a tool so how we go about this is up to you, but I'll do as much of the manual labor as is needed if you let me know how.

**Comment by Spartan97** [comment]
The tool won't be too difficult, but also probably not really worth our time currently. If you REALLY wanted to work for it you could go online using phpmyadmin and edit the individual rows... Would just require you to cross-reference the team IDs/names. And I'll need to give you write access if you want this.

**Comment by Spartan97** [comment]
ACTUALLY, it looks like sports-ref does have this data... Looks like all the boxscores have the stadium and time under the actual result. The requirement here though would be that we need to have a table identifying each team's home stadium by name.

Might actually just be easier to manually update since 95% of the games are home-site.

**Comment by jkeena88** [comment]
Yeah I can just manually update if you give me access. Does that require a SQL statement or can you just point/click/type in the phpmyadmin console?

**Comment by Spartan97** [comment]
It's all point and click. You'll select the table, then it should display all the rows, and you can just click a column and change the value.
